src/repository/scriptRepository.js

```diff
@@ -23,10 +40,20 @@ async function getScript(scriptDir, id) {
   return parseLines(file);
 }
 
+/**
+ * delete the specified script
+ * @param {string} id The script ID
+ * @return {Promise<*>} raw database result, unused
```

Comment:
If unused, should we not return anything? Or return true/false for success/failure?

Reply:
That's a good point. I had been thinking there might be a future situation where we would want to do something with the data returned from the database, but it would be returning the same information we already know. I'll convert this and the deleteActor function to return a boolean.